src/app/services/chat.service.ts

```diff
@@ -25,7 +25,7 @@ export class ChatService {
       {
         headers: {
           'Content-Type': 'application/json',
-          'User-Agent': 'OpenAPI-Generator/1.0/Javascript',
+          'X-User-Agent': 'OpenAPI-Generator/1.0/Javascript',
```

Comment:
Is it necessary?

Reply:
Not necessarily, but in console it was giving an error for unsafe usage on headers.